crypto/txscript/src/opcodes/mod.rs

```diff
@@ -883,12 +883,82 @@ opcode_list! {
             _ => Err(TxScriptError::InvalidSource("LockTimeVerify only applies to transaction inputs".to_string()))
         }
     }
+    opcode OpInputSpk<0xb2, 1>(self, vm) {
+        if vm.kip10_enabled {
+            match vm.script_source {
+                ScriptSource::TxInput{
+                    utxo_entry: kaspa_consensus_core::tx::UtxoEntry{
+                        script_public_key: spk @ kaspa_consensus_core::tx::ScriptPublicKey{
+                            version, ..},
+                        ..
+                    },
+                    ..
+                } => {
+                    let version = version.to_be_bytes();
```

Comment:
In most cases in consensus we use little-endian. Any specific reason to use big-endian?

Reply:
There are places that already uses be for version 
https://github.com/search?q=repo%3Akaspanet%2Frusty-kaspa%20version.to_be_bytes()&type=code

I want uniformity here